tests/test_solvermode/test_post_vector.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 
-@pytest.mark.setup
+@pytest.mark.settings_only
```

Comment:
This one isn't really `settings_only`, right? It is also reading a .dat file which is probably more "heavyweight"

Same for `test_change_create_mixture`?

Reply:
`test_post_vector` doesn't seem to require the `.dat.h5` file so I've changed it to use lightweight case read. I've also changed `test_change_create_mixture` to use lightweight case read, must have missed that one in the last PR.